content/intro-to-storybook/angular/en/simple-component.md

```diff
@@ -279,99 +258,57 @@ Make sure your components render data that doesn't change, so that your snapshot
-```
-
-Then create an `src/storybook.test.ts` file with the following in it:
-
-```typescript
-import * as path from 'path';
-import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';
-
-initStoryshots({
-  framework: 'angular',
-  configPath: path.join(__dirname, '../.storybook'),
-  test: multiSnapshotWithOptions(),
-});
+npm install -D @storybook/addon-storyshots babel-plugin-require-context-hook
 ```
 
-After that, create a `src/jest-config` folder with two files inside, `globalMocks.ts`:
+Update the `babel.config.js` that was created in the previous section to the following:
 
-```typescript
-const mock = () => {
-  let storage = {};
+```javascript
+module.exports = function(api) {
+  const isTest = api.env('test');
+  const presets = [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
+        },
+      },
+    ],
+    '@babel/preset-typescript',
+  ];
   return {
-    getItem: key => (key in storage ? storage[key] : null),
-    setItem: (key, value) => (storage[key] = value || ''),
-    removeItem: key => delete storage[key],
-    clear: () => (storage = {}),
+    presets,
+    plugins: isTest ? ['require-context-hook'] : [],
   };
 };
-
-Object.defineProperty(window, 'localStorage', { value: mock() });
-Object.defineProperty(window, 'sessionStorage', { value: mock() });
-Object.defineProperty(window, 'getComputedStyle', {
-  value: () => ['-webkit-appearance'],
-});
 ```
 
-and `setup.ts`:
+Then create the `src/storybook.test.js` file with the following in it:
 
 ```typescript
-import 'jest-preset-angular';
-import './globalMocks';
-```
-
-Then we need to add a new field to our `package.json`,
+import registerRequireContextHook from 'babel-plugin-require-context-hook/register';
+import initStoryshots from '@storybook/addon-storyshots';
 
-```json
-"jest": {
-    "coveragePathIgnorePatterns": [
-      "/jest-config/",
-      "/node_modules/"
-    ],
-    "preset": "jest-preset-angular",
-    "setupTestFrameworkScriptFile": "<rootDir>/src/jest-config/setup.ts",
-    "snapshotSerializers": [
-      "<rootDir>/node_modules/jest-preset-angular/AngularSnapshotSerializer.js",
-      "<rootDir>/node_modules/jest-preset-angular/HTMLCommentSerializer.js"
-    ],
-    "testPathIgnorePatterns": [
-      "/node_modules/",
-      "/build/",
-      "/storybook-static/"
-    ],
-    "transform": {
-      "^.+\\.(ts|js|html)$": "<rootDir>/node_modules/jest-preset-angular/preprocessor.js"
-    },
-    "moduleNameMapper": {
-      "\\.(css|less)$": "identity-obj-proxy"
-    }
-  },
+registerRequireContextHook();
+initStoryshots();
 ```
 
-a couple of new scripts to run `jest`
+Finally we need to make a small change in the `jest` key in our `package.json`:
 
 ```json
-"scripts": {
-  ...
-  "jest": "jest",
-  "jest:watch": "jest --watch"
-}
-```
-
-and update `src/tsconfig.app.json` to exclude `.test.ts` files
+{
+  ....
+   "transform": {
```

Comment:
It applies for the babel.config.js changes as well : as it makes sense for the react tutorial to update the jest configuration at this point of the tutorial as jest is included by default with CRA, i think these changes should be included into the 'Get Started' section directly for Angular.

Reply:
@geromegrignon if i'm reading this correctly, you're proposing that this should be moved into the `Get Started` section? If that's the case, i'm sorry but i have to disagree. As at that stage the addon `@storybook/addon-storyshots` is not present, and introducing it in there will cause issues, i thought about that and tried it and basically Jest threw a "fit" (pardon the bad pun). 
On a side note, as the team is pushing for mass adoption of a new way to write stories, with that  @tmeasday spearheaded the changes in the react tutorial. It's going to be updated incrementally